controlled-vocabulary/src/main/java/org/uniprot/cv/subcell/SubcellularLocationFileReader.java

```diff
@@ -200,57 +311,14 @@ private SubcellularLocationEntry parseSubcellularFileEntry(SubcellularFileEntry
                         .collect(Collectors.toList());
         retObj.synonymsSet(synList);
 
-        return retObj.build();
-    }
-
-    private List<SubcellularLocationEntry> updateListWithRelationShips(
-            List<SubcellularLocationEntry> list, List<SubcellularFileEntry> rawList) {
-        List<SubcellularLocationEntryBuilder> retList = new ArrayList<>(list.size());
-        for (SubcellularFileEntry raw : rawList) {
-            SubcellularLocationEntry targetFromList = findByIdentifier(list, getIdentifier(raw));
-            assert (targetFromList != null);
-            SubcellularLocationEntryBuilder target =
-                    SubcellularLocationEntryBuilder.from(targetFromList);
-
-            // Only check for those who have relationships
-            if (raw.hi.isEmpty() && raw.hp.isEmpty()) {
-                retList.add(target);
-                continue;
-            }
-
-            if (!raw.hi.isEmpty()) {
-                List<SubcellularLocationEntry> isA = new ArrayList<>();
-
-                for (String id : raw.hi) {
-                    isA.add(findByIdentifier(list, id));
-                }
-                target.isASet(isA);
-            }
-            if (!raw.hp.isEmpty()) {
-                for (String id : raw.hp) {
-                    target.partOfAdd(findByIdentifier(list, id));
-                }
-            }
-            retList.add(target);
-        }
-        return retList.stream()
-                .map(SubcellularLocationEntryBuilder::build)
-                .collect(Collectors.toList());
+        return (SubcellularLocationEntryImpl) retObj.build();
     }
 
     private String getIdentifier(SubcellularFileEntry raw) {
```

Comment:
Where are we using Map with SL-ID as key  in method parseSubcellularFileEntry? Are you suggesting to use SL-ID for map  `Map<String, SubcellularLocationEntryImpl> nameEntryMap` ? If yes, we cannot use SL-ID because the IS_A and Part_Of values in the input file has the name of the subcellular location.